source/common/network/happy_eyeballs_connection_impl.cc

```diff
@@ -16,6 +16,7 @@ HappyEyeballsConnectionImpl::HappyEyeballsConnectionImpl(
           {source_address, socket_factory, transport_socket_options, options}),
       next_attempt_timer_(dispatcher_.createTimer([this]() -> void { tryAnotherConnection(); })) {
   ENVOY_LOG(trace, "New connection.");
+  ENVOY_LOG_EVENT(debug, "happy_eyeballs_new_cx", "[C{}]", id_);
```

Comment:
You can use `ENVOY_CONN_LOG_EVENT` with a connection object so it prints out the `[C{id}]` formatting.